src/Cake.NuGet/NuGetModule.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Cake.Core.Composition;
+using Cake.Core.Packaging;
+
+namespace Cake.NuGet
+{
+    /// <summary>
+    /// The module responsible for registering
+    /// default types in the Cake.NuGet assembly.
+    /// </summary>
+    public sealed class NuGetModule : ICakeModule
```

Comment:
@patriksvensson  Sorry, this was the first/last comment that I made on my first brief pass through the PR.  I would have likely made the same comment on each of the modules.  I don't have any strong arguments either way on this particular naming, but it just sort of stood out when I was going through it.